docs/deprecations/migrating-from-prometheus-exec-to-prometheus.md

```diff
@@ -0,0 +1,28 @@
+# Removal of deprecated prometheusexec receiever
+## Why is this happening?
+There exist [security concerns](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/6722) using the promethusexec receiver.  All prometheus specific functionality should be supported in the "normal" prometheus (scraping) receiver, along with others
+
+## If I'm using the prometheusexec receiver, what should I do?
+
+We recommend you migrate your configuration to use the [prometheus](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/prometheusreceiver) opentelemetry receiver.
```

Comment:
The main reason someone would use the exec receiver is because it takes care of executing the exporter binary for them instead of needing to themselves. I think the node exporter is a standard example: https://prometheus.io/docs/guides/node-exporter/. Addressing this removal is more than updating their configuration to use the prometheus receiver. Believe Dmitry's commented to this effect as well